znai-reactjs/src/doc-elements/docMeta.js

```diff
@@ -37,4 +41,20 @@ function getDocId() {
     return docMeta.id
 }
 
-export {setDocMeta, isPreviewEnabled, getDocId}
+let supportLinkPromise = null;
+function getSupportLinkPromise() {
+    if (supportLinkPromise) {
+        return supportLinkPromise
+    }
+    const support = docMeta.support
+    if (support && support.link) {
+        supportLinkPromise = new Promise((resolve, reject) => resolve(support.link));
+    } else if (support && support.urlToFetchSupportLink) {
+        supportLinkPromise = jsonPromise(support.urlToFetchSupportLink + docMeta.id).then(supportMeta => supportMeta.link)
```

Comment:
I think we should not calculate url here and use url as is. You can do that magic on your side when you hijack doc meta.

Reply:
Deleted `+ docMeta.id`